app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -10,41 +11,32 @@ def self.ems_inv_to_hashes(ems, options = nil)
         new(ems, options).ems_inv_to_hashes
       end
 
-      def self.security_group_type
-        ManageIQ::Providers::Azure::CloudManager::SecurityGroup.name
-      end
-
       def initialize(ems, options = nil)
-        @ems                = ems
-        @config             = ems.connect
-        @subscription_id    = @config.subscription_id
-        @vmm                = ::Azure::Armrest::VirtualMachineService.new(@config)
-        @asm                = ::Azure::Armrest::AvailabilitySetService.new(@config)
-        @tds                = ::Azure::Armrest::TemplateDeploymentService.new(@config)
-        @vns                = ::Azure::Armrest::Network::VirtualNetworkService.new(@config)
-        @ips                = ::Azure::Armrest::Network::IpAddressService.new(@config)
-        @nis                = ::Azure::Armrest::Network::NetworkInterfaceService.new(@config)
-        @nsg                = ::Azure::Armrest::Network::NetworkSecurityGroupService.new(@config)
-        @rgs                = ::Azure::Armrest::ResourceGroupService.new(@config)
-        @sas                = ::Azure::Armrest::StorageAccountService.new(@config)
-        @options            = options || {}
-        @data               = {}
-        @data_index         = {}
-        @resource_to_stack  = {}
-        @network_interfaces = []
+        @ems               = ems
+        @config            = ems.connect
+        @subscription_id   = @config.subscription_id
+        # TODO(lsmola) NetworkManager, remove network endpoints once this is entirely moved under NetworkManager
+        @nis               = ::Azure::Armrest::Network::NetworkInterfaceService.new(@config)
+        @ips               = ::Azure::Armrest::Network::IpAddressService.new(@config)
```

Comment:
@bronaghs yes, there is still the Hardware/Network under the cloud. I am going to get rid of that, I will rewrite the google, then I will remove that section. Small steps :-)